pkg/apis/extensions/validation/validation.go

```diff
@@ -655,6 +656,33 @@ func validatePodSecurityPolicyVolumes(fldPath *field.Path, volumes []extensions.
 	return allErrs
 }
 
+const sysctlPatternSegmentFmt string = "([a-z0-9][_a-z0-9]*)?[a-z0-9*]"
+const sysctlPatternFmt string = "(" + apivalidation.SysctlSegmentFmt + "\\.)*" + sysctlPatternSegmentFmt
+
+var sysctlPatternRegexp = regexp.MustCompile("^" + sysctlPatternFmt + "$")
+
+func IsValidSysctlPattern(name string) bool {
```

Comment:
exported symbol => godocs, can you reuse the function from ``pkg/api/validation/validation.go``?